packages/vulcan-users/lib/modules/permissions.js

```diff
@@ -91,15 +91,30 @@ Users.getActions = user => {
 };
 
 /**
- * @summary check if a user is a member of a group
+ * @summary check if a user is at least member of one group
  * @param {Array} user
  * @param {String} group or array of groups
  */
-Users.isMemberOf = (user, groupOrGroups, document) => {
+Users.isAtLeastMemberOfOne = (user, groupOrGroups, document) => {
```

Comment:
@SachaG We want to express the fact that :
1) guest < member < admin, it's an ordered relationship. So we consider "members" to be "atLeast" "guests"
2) "ofOne" express the fact that you must match only one group to be allowed, not both (so ["entrepreneur", "social"] => "entrepreneur" and "social" and ["entrepreneur","social"] are all allowed